### PR TITLE
Support multilabel data in compute_accuracy

### PR DESCRIPTION
**Pull request by Bomme**

### CODE REVIEW

File: src/shared.py [hunk 1]
```diff
@@ -34,7 +36,10 @@ def load_id2path(index_file):
     fspec = open(index_file)
     id2path = dict()
     for line in fspec.readlines():
-        id, path, = line.strip().split("\t")
+        (
+            id,
+            path,
+        ) = line.strip().split("\t")
```
Comment (palonso) [inline src/shared.py]: I find this format for receiving two variables very difficult to read. 
Is `id, path = line.strip().split("\t")` ok to you?

File: test/test_shared.py [hunk 2]
```diff
@@ -35,3 +35,44 @@ def test_type_of_groundtruth():
     assert (
         shared.type_of_groundtruth(np.array([[0, 1], [1, 0]])) == "multiclass-indicator"
     )
+
+
+def test_compute_accuracy():
```
Comment (palonso) [inline test/test_shared.py]: Let's divide this into two tests
- `test_compute_accuracy_multiclass`
- `test_compute_accuracy_multilabel`

**Review by palonso — COMMENTED** [review]
Nice work! I left a couple of comments

File: src/shared.py (hunk 1 above)
Comment (Bomme) [reply src/shared.py]: I must have commited that by accident. black transformed this because there was a trailing comma after `path`

**Comment by palonso** [comment]
Great, thanks!

**Pull request merged**